etc/config-template.toml

```diff
@@ -118,6 +118,18 @@
 ## Attributes about this server, e.g. `{ zone = "us-west-1", disk = "ssd" }`.
 # labels = {}
 
+<<<<<<< HEAD
+=======
+## The working thread count of the background pool, which include the endpoint of and br, split-check,
+## region thread and other thread of delay-insensitive tasks.
+## The default value is 2 if the number of CPU cores is less than 16, otherwise 3.
+# background-thread-count = 2
+
+## If handle time is larger than the threshold, it will print slow log in endpoint.
+## The default value is 1s.
+# end-point-slow-log-threshold = "1s"
+
+>>>>>>> f9dca12f5... Add end_point_slow_log_threshold config (#9061)
```

Comment:
```suggestion
## If handle time is larger than the threshold, it will print slow log in endpoint.
## The default value is 1s.
# end-point-slow-log-threshold = "1s"

```

Reply:
Oh, I pushed the wrong commit...